tests/spec/support/examples/connection.rb

```diff
@@ -2,15 +2,15 @@
 
 # Examples for all adapter accessing the given base url
 shared_examples 'a connection making requests' do |base_url, adapter|
-  let(:conn) do
+  before :all do
```

Comment:
The names of the hooks have changed (the old names linger), [here are the RSpec hook names](https://relishapp.com/rspec/rspec-core/v/3-8/docs/hooks/before-and-after-hooks)

```suggestion
  before :suite do
```

Reply:
That raises a warning:

> WARNING: `before(:suite)` hooks are only supported on the RSpec configuration object. This `before(:suite)` hook, registered on an example group, will be ignored.

Based on [these hook docs](https://relishapp.com/rspec/rspec-core/v/2-2/docs/hooks/before-and-after-hooks#before/after-blocks-defined-in-config-are-run-in-order), the use of `before(:all)` looks correct.